singularity/debian/README_debian.md

```diff
@@ -16,25 +16,25 @@ cd grass
 __Build the singularity with__:
 
 ```bash
-sudo singularity build grass_development.simg singularity/debian/singularity_debian
+sudo singularity build grass_gis.simg singularity/debian/singularity_debian
 ```
 
 __To build a stable version__:
 
 change to the releasebranch or tag you want to build:
 
 ```bash
-git checkout remotes/origin/releasebranch_8_2
+git checkout 8.2.0
```

Comment:
Since 8.4 will be the new stable very soon, better use `8.4.0` here?

Reply:
Maybe I was not looking for underscores here.

Sure, between 8.2 and 8.4, this PR is trying to update to 8.4.

However, better than updating is to remove these references which needed to be updated every time the number changes. Including micro version (8.2.0) creates even more need for updates than just the branch number (8_2): every release (multiple times a year) as opposed to every major or minor (once a year or so).

Any suggestions for removing the need to update? My suggestions:

```bash
git checkout 3.4.0
git checkout {version}
git checkout {version number or branch name}
git checkout {tag or branch}
```

As for the desired frequency of updates, note that this PR is on main updating from 8.2.